azkaban-common/src/main/java/azkaban/imagemgmt/permission/PermissionManagerImpl.java

```diff
@@ -76,4 +83,24 @@ public boolean hasPermission(final String imageTypeName, final String userId, fi
 
     return hasPermission;
   }
+
+  /**
+   * Checks if the user has permission to manage high priority flows.
+   * @param userId : must not be null.
+   * @return boolean
+   */
+  @Override
+  public boolean hasPermission(final String userId) {
+    requireNonNull(userId, "userId must not be null");
+    if (this.hpFlowDao.isHPFlowOwner(userId)) {
```

Comment:
Who has permission to add username in hpflowowner? Please add that detail in comment for the method.

Reply:
Just like ImageManagement, for this, the ability is with Azkaban admins, I will write a comment explaining that.